homeassistant/components/switchbot/sensor.py

```diff
@@ -126,6 +126,10 @@
         device_class=SensorDeviceClass.ENUM,
         options=HumidifierWaterLevel.get_levels(),
     ),
+    "battery_range": SensorEntityDescription(
+        key="battery_range",
+        translation_key="battery_range",
+    ),
```

Comment:
Okay so you explained me in a different PR that this value can be `>=something%`. What options are there? Should we map them instead to `low`, `medium`, `high`? We can declare possible values in the `options` field and make it an enum sensor